device_service.go

```diff
@@ -14,10 +14,10 @@ const (
 )
 
 type Device struct {
-	Id       int     `json:"id"`
-	Name     string  `json:"name"`
-	Interval int     `json:"interval"`
-	Value    float64 `json:"value"`
+	Id       int     `json:"id" bson:"id"`
```

Comment:
no need to change it of course, but golang way would be to capitalize field name (`ID int`). we're probably not very consistent in our projects anyway...

Reply:
oh yeah, there's one thing wrong here. it should be:

```suggestion
	ID       primitive.ObjectId     `json:"id" bson:"_id"`
```

look, you're entries in DB have 2 ids:

```
{ "_id" : ObjectId("5ff881247e71d261b47547a2"), "id" : 0, "name" : "dev1", "interval" : 3, "value" : 7 }
```